### DISCUSSION TITLE
BRY: Exception> 'value_error' - Missing arguments, remaining type 'b'

### DISCUSSION THREAD

**Discussion by Slyke** [arendst/Tasmota#14396] [Q&A] [answered]
Following along this documentation: https://tasmota.github.io/docs/LVGL/#touch-screen-support

I get to the point of creating the colorwheel:
```
colp = lv.colorwheel(scr)
colp.set_size(130, 130)
colp.set_pos(10,30)
```

When I run this in the Berry console however it errors with:
```
load("autoexec")
true
colp = lv.colorwheel(scr)
colp.set_size(130, 130)
colp.set_pos(10,30)
BRY: Exception> 'value_error' - Missing arguments, remaining type 'b'
stack traceback:
	<native>: in native function
	<unknown source>: in function `main`
```

I do see the results on the screen of the 3 buttons, but obviously not the color wheel.

**Comment by s-hadinger** [comment]
I don't have a device at hand, I will test later. Can you please confirm the Tasmota version you are using?

Edit: I found the version in the former issue.

**Comment by s-hadinger** [comment, marked as answer]
The constructor for `colorwheel` changed in LVGL8 and now takes a boolean `knob_recolor` as per LVGL documentation: https://docs.lvgl.io/master/widgets/extra/colorwheel.html

You should do the following:

```
colp = lv.colorwheel(scr, false)
colp.set_size(130, 130)
colp.set_pos(10,30)
```

Let me fix the documentation accordingly

**Reply by Slyke** [reply]
Thank you! Will give a shot soon.
